spinnaker/client.go

```diff
@@ -351,6 +352,26 @@ func (c *client) GetPipelineConfig(app, pipelineConfigID string) (*PipelineConfi
 	return &config, nil
 }
 
+func (c *client) ListPipeline(app string) ([]PipelineConfig, error) {
```

Comment:
Should probably rename to `ListPipelineConfigs`, as well as the associated action.